includes/class-omise-money.php

```diff
@@ -48,6 +48,20 @@ public static function to_subunit( $amount, $currency ) {
 		return $amount * self::$subunit_multiplier[ $currency ];
 	}
 
+	/**
+	 * convert omise amount to human readable amount.
+	 */
+	public static function to_readable( $amount, $currency ) {
```

Comment:
`to_readable` can imply that the amount is in some unreadable format. I would suggest to change the name `to_readable` to something like `convertDenomination` or `convertCurrencyUnit`  or `unitConversion`.